src/mobile/components/layout/templates/main.jade

```diff
@@ -7,11 +7,11 @@ block body
       block banner
         mixin login-signup("header")
       header
-        != stitch && stitch.components.NavBar({ user: sd.CURRENT_USER, notificationCount: sd.NOTIFICATION_COUNT })
+        != stitch && stitch.components.NavBar({ user: sd.CURRENT_USER, flashMessage: sd.flashMessage, notificationCount: sd.NOTIFICATION_COUNT })
```

Comment:
Wondering what the difference is between this and index.jade, and why we have 2 navbars in this file...

Reply:
I believe I am responsible for the double-header. 😅 (https://github.com/artsy/force/pull/5261)

I think what you have is fine, but note that the sticky-header behavior on mobile web for non-reaction pages is a little weird. I would verify that this looks fine on the home page (logged in _and_ logged out) and on the artist page (logged in/out) to be sure that nothing's messed up!